app/services/hyrax/actor_factory.rb

```diff
@@ -9,9 +9,10 @@ def self.stack_actors(curation_concern)
        Hyrax::Actors::AttachFilesActor,
        Hyrax::Actors::ApplyOrderActor,
        Hyrax::Actors::InterpretVisibilityActor,
+       DefaultAdminSetActor,
+       Hyrax::Actors::InitializeWorkflowActor,
```

Comment:
The order of actors in Hyrax is different from the order of actors in Sufia. See https://github.com/projecthydra/sufia/blob/master/app/services/sufia/actor_factory.rb. I went with the order from Sufia where I've seen workflow working before rather than Hyrax. Do you think Sufia is currently wrong too?